cli/options/options.go

```diff
@@ -288,6 +289,25 @@ func HandleLoggingParams(debug bool, cfg config.ApplicationConfiguration) (*zap.
 	return log, &cc.Level, _winfileSinkCloser, err
 }
 
+// GetRPCWithActor returns an RPC client instance and Actor instance for the given context.
+func GetRPCWithActor(gctx context.Context, ctx *cli.Context, signers ...actor.SignerAccount) (*rpcclient.Client, *actor.Actor, cli.ExitCoder) {
+	if len(signers) == 0 {
```

Comment:
We need one more tiny change in the method's signature to match the `GetRPCWithInvoker`. Make `signers` argument non-variadic, just a simple list. And also you can safely remove `len(signers) == 0` check, because Actor's constructor already has it.